lib/ansible/modules/apt.py

```diff
@@ -1292,10 +1301,16 @@ def main():
             module.warn("Auto-installing missing dependency without updating cache: %s" % apt_pkg_name)
         else:
             module.warn("Updating cache and auto-installing missing dependency: %s" % apt_pkg_name)
-            module.run_command(['apt-get', 'update'], check_rc=True)
+            module.run_command([APT_GET_CMD, 'update'], check_rc=True)
 
         # try to install the apt Python binding
-        module.run_command(['apt-get', 'install', '--no-install-recommends', apt_pkg_name, '-y', '-q'], check_rc=True)
+        apt_pkg_cmd = [APT_GET_CMD, 'install', apt_pkg_name, '-y', '-q', dpkg_options]
+        if install_recommends:
+            apt_pkg_cmd.append('--install-recommends')
+        else:
+            apt_pkg_cmd.append('--no-install-recommends')
```

Comment:
for consistency, it could it fall back to the configured/OS default like `install()`
```python
        if install_recommends is False:
            cmd += " -o APT::Install-Recommends=no"
        elif install_recommends is True: 
            cmd += " -o APT::Install-Recommends=yes"
        # install_recommends is None uses the OS default
```